docs/source/guides/libraries/swift_ios.rst

```diff
@@ -0,0 +1,282 @@
+-	Xcode 9.2
+-	carthage 0.29.0
+-	cmake 3.11.0
+-	iPhone 7 iOS 11.2 Simulator
+ 
+Hyperledger Iroha iOS library
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Iroha has the following features:
+
+1.	Creation and management of custom complex assets, such as currencies or indivisible rights, serial numbers, patents, etc.
+2.	Management of user accounts
+3.	Taxonomy of accounts based on domains — or sub-ledgers in the system
+4.	The system of rights and verification of user permissions for the execution of transactions and queries in the system
+5.	Validation of business rules for transactions and queries in the system
+
+Among the non-functional requirements can be noted a high degree of network fault tolerance (Byzantine Fault Tolerant).
+Iroha iOS library gives the ability to provide key generation and signing logic for queries and transactions passed to Iroha blockchain.
+Let's start with the detailed instructions how to install Iroha on the local machine.
+
+
+ 
+Instruction
+^^^^^^^^^^^
+
+1.	Open the terminal and go to the folder where You want to install all artifacts:
+
+  .. code-block:: bash
+
+      cd path/to/your/folder/for/example/iroha-ios/project/
+
+2.	Clone the repository for the iOS client:
+
+  .. code-block:: bash
+
+      git clone https://github.com/soramitsu/iroha-ios.git
+
+3.	Go to the Iroha-ios folder:
+
+  .. code-block:: bash
+
+      cd iroha-ios/
+
+4.	Update dependencies:
+
+  .. code-block:: bash
+
+      carthage update --platform iOS
+
+5.	Go to sample project directory:
+
+  .. code-block:: bash
+
+      cd SwiftyIrohaExample
+
+6.	Update dependencies for the sample:
+
+  .. code-block:: bash
+
+      carthage update --platform iOS
+
+7.	Go to GRPC library source's location:
+
+  .. code-block:: bash
+
+      cd grpc-swift/
+
+8.	Remove old library sources:
+
+.. note:: Make sure You've located in ``grpc-swift/`` subfolder
+
+  .. code-block:: bash
+
+      # removes all files from the current directory
+      rm -rf ./*
+      #removes all hidden files too (so clean build can be done)
+      rm -rf ./.*
+ 
+9.	Download release version of GRCP from git to the current directory:
+
+  .. code-block:: bash
+
+      git clone --branch 0.3.3 https://github.com/grpc/grpc-swift.git .
+
+10.	Build library:
+
+  .. code-block:: bash
+
+      make
+
+11.	Go to the root of your playground folder (from the first step - path/to/your/folder/for/example/iroha-ios/project/):
+
+  .. code-block:: bash
+
+      cd ../../..
+
+.. note:: Make sure now You are located in ``path/to/your/folder/for/example/iroha-ios/project/`` folder
+
+12.	This step downloads script for client library which is needed to build client library. Clone it from the repository:
+
+  .. code-block:: bash
+
+      curl https://raw.githubusercontent.com/hyperledger/iroha/master/shared_model/packages/ios/ios-build.sh > ios-build.sh
+
+13.	Optional step. If You have issues with cloning during ios-build.sh execution do the following command before the script invocation:
```

Comment:
What do you think about that?:

If You have issues with cloning during ios-build.sh execution do the following command before the script invocation
->
If there are issues with git cloning at ios-build.sh then the following command may help